program/src/process_management.rs

```diff
@@ -8,14 +8,15 @@ use solana_program::{account_info::AccountInfo, entrypoint::ProgramResult, msg,
 
 use crate::logic::check_rent_exempt;
 use crate::processor::StakeType;
+use crate::state::Lido;
```

Comment:
The changes here look innocent to me, but if possible I would avoid changing things in `program`, so we don’t accidentally introduce a difference between the currently deployed program and the source code here.

Reply:
Yeah, in this case  we weren't exporting the `logic` module, I thought was simpler to move the function to Lido's state